### PR TITLE
UTC-176-C35 Six compound tone diacritics

### PR DESCRIPTION
**Pull request by macchiati**
[[176-C35](https://www.unicode.org/cgi-bin/GetL2Ref.pl?176-C35)] Consensus: Provisionally assign U+1AD0..U+1AD5 for six compound tone diacritics, as given in [L2/23-188](https://www.unicode.org/cgi-bin/GetMatchingDocs.pl?L2/23-188). (Reference: Section 7 of [L2/23-164](https://www.unicode.org/cgi-bin/GetMatchingDocs.pl?L2/23-164))

[[181-C60](https://www.unicode.org/cgi-bin/GetL2Ref.pl?181-C60)] Consensus: UTC accepts for encoding in Unicode Version 17.0 the following 297 Arabic, Bengali, Han, Kannada, Latin, Oriya, phonetic, Sharada, Tangut, Telugu, and symbol characters for which the code points have previously been provisionally assigned:

        Arabic (40 characters): 088F, FBC3..FBD2, FD90..FD91, FDC8..FDCE, 10EC5..10EC7, 10ED0..10ED8, 10EFA..10EFB
        Bengali (1 character): 09FF
        Han (5 characters): 16FF2..16FF6
        Kannada (1 character): 0CDC
        Latin (2 characters): A7D2, A7D4
        Oriya (2 characters): 0B53..0B54
        Phonetic (30 characters): 1ACF, 1AD0..1ADD, 1AE0..1AEB, A7CE..A7CF, A7F1
        Sharada (8 characters): 11B60..11B67
        Tangut (145 characters): 187F8..187FF, 18D09..18D1E, 18D80..18DF2
        Telugu (1 character): 0C5C
        Symbols (62 characters): 2B96, 1CCFA..1CCFC, 1CEBA..1CED0, 1CEE0..1CEF0, 1F777..1F77A, 1F8D0..1F8D8, 1FA54..1FA57, 1FBFA

* https://github.com/unicode-org/sah/issues/298
* https://github.com/unicode-org/utc-release-management/issues/55

### CODE REVIEW

**Comment by macchiati** [comment]
Thanks, @eggrobin 
What do you think about changing https://github.com/unicode-org/unicodetools/blob/main/docs/pipeline.md to be more explicit:

OLD
 pipeline-* — Set label
NEW
 pipeline-* — Set label to pipeline-provisionally-assigned or pipeline-<version>, depending on whether provisional or not.

**Comment by macchiati** [comment]
Hmmm. The description does include a link to a UTC decision, namely https://www.unicode.org/L2/L2023/23157.htm#176-C35, but the Pipeline / UTC decision check fails.

**Comment by eggrobin** [comment]
> Hmmm. The description does include a link to a UTC decision, namely https://www.unicode.org/L2/L2023/23157.htm#176-C35, but the Pipeline / UTC decision check fails.

Yeah this regex is too strict, it expects the UTC- prefix: https://github.com/unicode-org/unicodetools/blob/main/py%2Fpipeline-workflow%2Fcheck-utc-decision.py#L10

**Comment by macchiati** [comment]
Thanks!

File: unicodetools/data/ucd/dev/auxiliary/LineBreakTest.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 <body bgcolor='#FFFFFF'>
 <h2>Line_Break Chart</h2>
 <p><b>Unicode Version:</b> 16.0.0</p>
-<p><b>Date:</b> 2023-11-03, 21:06:18 GMT</p>
+<p><b>Date:</b> 2023-12-20, 22:58:48 GMT</p>
```
Comment (markusicu) [inline unicodetools/data/ucd/dev/auxiliary/LineBreakTest.html:10]: In the future, please try to exclude files with only time stamp changes. They just increase the surface for merge conflicts.

**Review by markusicu — COMMENTED** [review]
lgtm

**Review by markusicu — COMMENTED** [review]
actually, looking at https://github.com/unicode-org/unicodetools/pull/588, these should also be Diacritic in PropList.txt
@eggrobin FYI

**Comment by eggrobin** [comment]
Yes, I think this is a fairly clear-cut case of « diacritics are diacritics ».

(For, in contrast, a case where the Diacritic property is not clear-cut, see the very fun discussion in unicode-org/properties#233.)

**Comment by eggrobin** [comment]
In January markus wrote:
> actually, looking at https://github.com/unicode-org/unicodetools/pull/588, these should also be Diacritic in PropList.txt

(and then Mark sat on it). I’ll go ahead and do it.

**Review by markusicu — APPROVED** [review]

**Pull request merged**